asapdiscovery-dataviz/asapdiscovery/dataviz/gif_viz.py

```diff
@@ -140,7 +146,7 @@ def write_traj_visualizations(self):
             output_paths.append(output_path)
         return output_paths
 
-    def write_traj_visualization(self, traj, system, path):
+    def write_traj_visualization(self, traj, system, path, bool_static_view_only):
```

Comment:
Can we change the name of this variable to just `static_view_only`

Reply:
done